test/jdk/com/sun/crypto/provider/Cipher/AEAD/GCMBufferTest.java

```diff
@@ -128,7 +129,8 @@
     GCMBufferTest(String algo, List<dtype> ops) {
         this.algo = algo;
         this.ops = ops;
-        theoreticalCheck = true;
+        theoreticalCheckUpdate = false;
```

Comment:
Please add a comment indicating the tests which use this variable are being disabled.

Reply:
Shouldn't the changes being proposed satisfy the tests?

Reply:
The reason I edited this test separating the check for update and final is because according to the [java documentation](https://download.java.net/java/early_access/jdk16/docs/api/java.base/javax/crypto/Cipher.html#update(byte%5B%5D,int,int,byte%5B%5D)), `cipher.update` must return `the number of bytes stored in output`. This number, in our case, is zero given that we just stored the plain text in a buffer and did not encrypt anything yet. However, other implementations can return a different number here.

Before my changes, the `theoreticalCheck` was already there ([here](https://github.com/samasri/openj9-openjdk-jdk/blob/ac04db2e39441f285acebb657aa388043a894770/test/jdk/com/sun/crypto/provider/Cipher/AEAD/GCMBufferTest.java#L64-L65)) because it was expected that in some cases, the theoretical check is not straight forward to verify. I am turning off the check for update here because our implementation is complying with the cipher.update expectations in the docs, but is different than what other implementations return.

Reply:
If the test is modified, an IBM copyright must be added to this file.

Reply:
Done.

Reply:
Done.

Reply:
The copyright should list two dates (even when they're the same):
```
* (c) Copyright IBM Corp. 2021, 2021 All Rights Reserved
```

Reply:
> Please add a comment indicating the tests which use this variable are being disabled.

I don't see the comment Peter requested.